exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,11 @@
 function halve(number) {
   // complete the function here
+  const halveNumber = number / 2;
+  return halveNumber;
```

Comment:
If you wanted to reduce the number of lines you could return the expression without staring it in a variable: `return number/2`

Reply:
Yeah, I just tried to write code clear